src/update.c

```diff
@@ -476,7 +474,17 @@ static int main_update()
 
 	timelist_timer_start(global_times, "Create Update List");
 
-	updates = create_update_list(server_manifest);
+	/* When the minversion is 0 (set to 0 when missing from MoM header),
+	   delta manifests will not be used and the old update list implementation
+	   will be used. When using the new update list implementation if a
+	   minversion update occurred without a minversion header, all files without
+	   a content change would download fullfiles. */
+	if (server_manifest->minversion == 0) {
```

Comment:
Mixer has included the minversion header field in the MoM for a while now. Are there any arguments for keeping this special case or can I consolidate these?

Reply:
If we can remove the call to link_manifests, update performance could be improved and several delta manifest corner cases could be eliminated.